packages/zoe/tools/scriptedPriceAuthority.js

```diff
@@ -89,6 +102,7 @@ export function makeScriptedPriceAuthority(options) {
     updateState: t => {
       t = TimeMath.absValue(t);
       currentPrice =
+        // @ts-expect-error XXX: this does not support quoteInterval being a RelativeTimeRecord
         priceList[Number(Number(t / quoteInterval) % priceList.length)];
```

Comment:
There should be some kind of runtime change here. This shouldn't be working now. Maybe unused?